README.md

```diff
@@ -32,12 +32,59 @@ Build Requirements
 
 Sccache is a [Rust](https://www.rust-lang.org/) program. Building it requires `cargo` (and thus `rustc`). sccache currently requires **Rust 1.21**.
 
-We recommend you install Rust via [Rustup](https://rustup.rs/). The generated binaries can be built so that they are very portable, see [scripts/build-release.sh](scripts/build-release.sh). By default `sccache` supports a local disk cache. To build `sccache` with support for `S3` and/or `Redis` cache backends, add `--features=all` or select a specific feature by passing `s3`, `gcs`, and/or `redis`. Refer the [Cargo Documentation](http://doc.crates.io/manifest.html#the-features-section) for details.
+We recommend you install Rust via [Rustup](https://rustup.rs/). The generated binaries can be built so that they are very [portable](#building-portable-binaries)). By default `sccache` supports a local disk cache. To build `sccache` with support for `S3` and/or `Redis` cache backends, add `--features=all` or select a specific feature by passing `s3`, `gcs`, and/or `redis`. Refer the [Cargo Documentation](http://doc.crates.io/manifest.html#the-features-section) for details.
 
-## Build
+Build
+-----
 
 > $ cargo build [--features=all|redis|s3|gcs] [--release]
 
+### Building portable binaries
+
+When building with the `gcs` feature, `sccache` will depend on OpenSSL, which can be an annoyance if you want to distribute portable binaries. It is possible to statically link against OpenSSL using the steps below before building with `cargo`.
+
+#### Linux
+
+You will need to download and build OpenSSL with `-fPIC` in order to statically link against it.
```

Comment:
We might want to recommend building with the [`rust-musl-builder`](https://github.com/emk/rust-musl-builder/) Docker image, which is what I was previously using to build release binaries:
https://github.com/mozilla/sccache/blob/1a82a4783d360adb6fad2ede3d14c80b177912c4/scripts/build-release.sh